### ISSUE TITLE
Cocoapods problem

### ISSUE THREAD

**Issue by rvsrvs** [thoughtbot/Argo#37]
I used cocoapods to fetch Argo.  It has a dependency on Runes in the JSONOperators file.  But Runes doesn't have a Podspec so Argo won't compile..

**Comment by gfontenot** [comment]
Runes isn't in CocoaPods yet, but the version of Argo in the podspec  (0.2.2) doesn't require Runes. If you use that version, you shouldn't have any issues.

**Comment by gfontenot** [comment]
We will update the podspec once we add Runes to CocoaPods, but until then the podspec will be static.

**Comment by rvsrvs** [comment]
Guess I'm not understanding your comment.  
Following the instructions on the Argo main page in github and adding:

pod 'Argo', :git => 'https://github.com/thoughtbot/Argo'

to my podfile got me a JSONOperators.swift file that explicitly used Runes. But doing the pod update with that line in my podfile did not get any Runes files..    Should I be doing something different?

**Comment by gfontenot** [comment]
Honestly, I kind of regret adding this to CocoaPods before Swift support made it into the release version. I always forget how CocoaPods works when you point it at a specific git source. Sorry for your frustration.

I _think_ what you need to do is add a specific tag to your podfile:

```
pod 'Argo', :git => 'https://github.com/thoughtbot/Argo', :tag => 'v0.2.2'`
```

I haven't verified that, but if that works I'll make sure to add it to the README.

**Comment by rvsrvs** [comment]
No problem at all.

actually just adding 

pod ‘Argo’

works and pulls the 0.2 tag which doesn’t have the Runes dependency.  Thing is, Runes is kind of handy and I’m working on pulling it now :)

BTW, I notice you guys have a presence in Boston.  But I hadn’t been aware of you before.  Always good to hear from local iOS guys.

rvs

**Comment by gfontenot** [comment]
Ah, yeah, there _is_ an "official" version in CocoaPods Trunk, but we aren't updating it yet. I think there were some fixes between 0.2 and 0.2.2 that you might find useful, but at least you're getting it installed ok.

We do have a presence in Boston! We actually [run a monthly meetup](http://www.meetup.com/NSCoder-Night-Boston/) if you ever want to drop by.
